packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -5,72 +5,80 @@ title: Role Assignment
+| `GROUP_MEMBERSHIP_ADMIN`                | Group membership administrator      | [Groups](/docs/reference/api/groups/) |
+| `HELP_DESK_ADMIN`                       | Help desk administrator             | [Groups](/docs/reference/api/groups/) |
+| `MOBILE_ADMIN`                          | Mobile administrator                |                                       |
+| `ORG_ADMIN`                             | Organizational administrator        |                                       |
+| `READ_ONLY_ADMIN`                       | Read-only administrator             |                                       |
+| `REPORT_ADMIN`                          | Report administrator                |                                       |
+| `SUPER_ADMIN`                           | Super administrator                 |                                       |
+| `USER_ADMIN`                            | Group administrator                 | [Groups](/docs/reference/api/groups/) |
 
 ### IAM-based standard role Types
 <ApiLifecycle access="ea" />
-We also support the following IAM-based standard roles:
+
+Okta also supports the following IAM-based standard roles:
 | Role type                                                  | Label                               | Permissions                                   |
 | :--------------------------------------------------------- | :---------------------------------- | :-------------------------------------------- |
-| `ACCESS_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | Access Certifications Administrator | `okta.governance.accessCertifications.manage` |
-| `ACCESS_REQUESTS_ADMIN`       <ApiLifecycle access="ea" /> | Access Requests Administrator       | `okta.governance.accessRequests.manage`       |
-IAM-based standard roles could be assigned using [standard](#standard-role-assignment-steps) or [custom role](#custom-role-assignment) assignment operations.
-Similar to other standard roles, these roles are also immutable and can't be updated or deleted.
+| `ACCESS_CERTIFICATIONS_ADMIN` <ApiLifecycle access="ea" /> | Access certifications administrator | `okta.governance.accessCertifications.manage` |
+| `ACCESS_REQUESTS_ADMIN`       <ApiLifecycle access="ea" /> | Access requests administrator       | `okta.governance.accessRequests.manage`       |
+
+You can assign IAM-based standard roles using [standard](#standard-role-assignment-steps) or [custom role](#custom-role-assignment) assignment operations. These roles are immutable and can't be updated or deleted.
 
 ### Standard Role Assignment steps
-Perform standard role assignment in two steps:
 
-1. Assign a role to a user or group. At this point, the admin has the supported privileges of the role over all resources across organization.
+Perform a standard role assignment in two steps:
+
+1. Assign a role to a user or group. At this point, the admin has the supported privileges of the role over all resources across the organization.
```

Comment:
```suggestion
1. Assign a role to a user or group. At this point, the admin has the supported privileges of the role over all resources across the org.
```

Reply:
Is this true? Only super admins have access to all resources. Maybe "Assign a role to a user or group. The user now has access to the admin permissions and resources for the role."